File: Tickets-backend/Data/DTO/ReadTicket.cs
using System.ComponentModel.DataAnnotations;

namespace Tickets.Data.DTO
{
    public class ReadTicket
    {
        public int Id { get; set; }   
        public int Quantidade { get; set; }        
        public char Situacao { get; set; }
        public DateTime DataEntrega { get; set; }
    }
}

File: Tickets-Frontend/Data/Dto/ReadTicket.pas
unit ReadTicket;

interface

type
  TReadTicket = class
  private
    FId: Integer;
    FQuantidade: Integer;
    FSituacao: Char;
    FDataEntrega: TDateTime;
  public
    property Id: Integer read FId write FId;
    property Quantidade: Integer read FQuantidade write FQuantidade;
    property Situacao: Char read FSituacao write FSituacao;
    property DataEntrega: TDateTime read FDataEntrega write FDataEntrega;
  end;


implementation

end.
